### PR TITLE
Allow configure posenet image size.

### PR DESCRIPTION
**Pull request by pgrill**
https://lanthorn.atlassian.net/browse/LP-370

### CODE REVIEW

File: libs/detectors/edgetpu/posenet.py [hunk 1]
```diff
@@ -56,14 +57,14 @@ def inference(self, resized_rgb_image):
         Returns:
             result: a dictionary contains of [{"id": 0, "bbox": [x1, y1, x2, y2], "score":s%}, {...}, {...}, ...]
         """
-        assert resized_rgb_image.shape == (721, 1281, 3)
+        assert resized_rgb_image.shape == (self.h, self.w, 3)  # noqa
```
Comment (renzodgc) [inline libs/detectors/edgetpu/posenet.py]: What does #noqa stands for?

**Review by renzodgc — APPROVED** [review]

File: libs/detectors/edgetpu/posenet.py (hunk 1 above)
Comment (pgrill) [reply libs/detectors/edgetpu/posenet.py]: `#noqa` is used to notify flake8 that the line should be ignored.

However, in that case is not required. I removed the comment.

**Review by renzodgc — APPROVED** [review]

File: config-coral.ini [hunk 2]
```diff
@@ -58,8 +58,8 @@ Device = EdgeTPU
 ; posenet use Coral's pose estimator to extract body's bounding boxes.
 Name = posenet
 ;ImageSize should be 3 numbers seperated by commas, no spaces: 300,300,3
-;For PoseNet model use: 1281,721,3
-ImageSize = 1281,721,3
+;For PoseNet model you can use: [1281,721,3], [641,481,3] or [481,353,3]
+ImageSize = 641,481,3
```
Comment (mats-claassen) [inline config-coral.ini:62]: Do you think this size is a better default than the original?

**Review by mats-claassen — APPROVED** [review]

File: config-coral.ini (hunk 2 above)
Comment (pgrill) [reply config-coral.ini:62]: When I tested the posnet model with the original size (1281,721,3) I wasn't able to have a fluent live-feed.
With this size the live-feed works correctly and the accuracy is still very hight.

Comment (renzodgc) [reply config-coral.ini:62]: So the model with the biggest input is the fastest? :thinking:

Comment (pgrill) [reply config-coral.ini:62]: No, the model with the biggest size is the slowest, but it has better accuracy.

**Pull request merged**